navit/navit_shipped_gtkgui.xml

```diff
@@ -0,0 +1,468 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
```

Comment:
use xslt to generate a config derived from out main shipped config, don't add a new one